.github/workflows/ruby-build.yml

```diff
@@ -202,13 +203,14 @@ jobs:
           echo 'name: sample-tests
           version: 0.0.0
           dependencies:
-            codeql/ruby-all: 0.0.1
+            codeql/ruby-all: "*"
           extractor: ruby
           tests: .
           ' > qlpack.yml
       - name: Run QL test
         shell: bash
         run: |
+          codeql pack install .
```

Comment:
I think this is not needed. All packs should be present in the "ruby-bundle"; no need to install things from the registry.
```suggestion
```